chemprop/nn/message_passing/base.py

```diff
@@ -176,7 +176,9 @@ def finalize(self, M: Tensor, V: Tensor, V_d: Tensor | None) -> Tensor:
                 H = self.W_d(torch.cat((H, V_d), dim=1))  # V x (d_o + d_vd)
                 H = self.dropout(H)
             except RuntimeError:
-                raise InvalidShapeError("V_d", V_d.shape, [len(H), self.W_d.in_features])
+                raise InvalidShapeError(
+                    "V_d", V_d.shape, [len(H), self.W_d.in_features - M.shape[1]]
```

Comment:
```suggestion
                    "V_d", V_d.shape, [len(H), self.W_d.in_features - self.W_o.out_features]
```
Thinking about it some more, I think using `W_o` to get the dimension we want is clearer. While M.shape[1] gives the right dimension, you'd have to go see where `M` is defined. In this function we have a comment that says the output of `W_o` is `V x d_o`. What do you think?